tests/integration/upgrades/test_small_deployment_upgrades.py

```diff
@@ -245,11 +246,13 @@ async def test_upgrade_rollback_from_local(
         )
 
         logger.info(f"Rolling back to {version}")
+        # TODO: return to 2/edge channel instead once this channel's latest 2.17 charm
+        # revision points to snap rev. 65 instead of snap rev. 62.
```

Comment:
This is going to be rolled back once this change with snap rev. 65 has been merged. Keeping it for this PR so we can be sure upgrade tests are working now.